{
  Function<Exception,Object> function=new ReturnNullOn303();
  HttpResponse response=EasyMock.createMock(HttpResponse.class);
  HttpResponseException exception=EasyMock.createMock(HttpResponseException.class);
  expect(response.getStatusCode()).andReturn(600);
  expect(exception.getResponse()).andReturn(response);
  expect(exception.getResponse()).andReturn(response);
  expect(exception.getCause()).andReturn(null);
  replay(response);
  replay(exception);
  try {
    function.apply(exception);
  }
 catch (  Exception ex) {
    assertEquals(ex,exception);
  }
  verify(response);
  verify(exception);
}
